<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_skybox">
    <property name="title">
     <string>skybox</string>
    </property>
    <addaction name="open_skybox_skb"/>
    <addaction name="open_skybox"/>
    <addaction name="open_skybox_textures"/>
    <addaction name="separator"/>
    <addaction name="load_default"/>
    <addaction name="separator"/>
    <addaction name="actionclear_skybox"/>
   </widget>
   <widget class="QMenu" name="menu_obj">
    <property name="title">
     <string>obj</string>
    </property>
    <addaction name="load"/>
    <addaction name="separator"/>
    <addaction name="create_squirrel"/>
    <addaction name="create_cube"/>
    <addaction name="create_monkey"/>
    <addaction name="separator"/>
    <addaction name="actiontest_create_objects"/>
    <addaction name="separator"/>
    <addaction name="actionclear_objects"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_skybox"/>
   <addaction name="menu_obj"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="object_info">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="object_list">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="control_info">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Move: W,A,S,D&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Rotate: Q,E&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionopen_obj">
   <property name="text">
    <string>open *.obj</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="open_skybox_skb">
   <property name="text">
    <string>open ( *.skb )</string>
   </property>
  </action>
  <action name="open_skybox">
   <property name="text">
    <string>open ( one texture )</string>
   </property>
  </action>
  <action name="open_skybox_textures">
   <property name="text">
    <string>open ( six textures )</string>
   </property>
  </action>
  <action name="load_default">
   <property name="text">
    <string>create default</string>
   </property>
  </action>
  <action name="load">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="create_squirrel">
   <property name="text">
    <string>create squirrel</string>
   </property>
  </action>
  <action name="create_cube">
   <property name="text">
    <string>create cube</string>
   </property>
  </action>
  <action name="create_monkey">
   <property name="text">
    <string>create monkey</string>
   </property>
  </action>
  <action name="actionclear_objects">
   <property name="text">
    <string>clear objects</string>
   </property>
  </action>
  <action name="actionclear_skybox">
   <property name="text">
    <string>clear skybox</string>
   </property>
  </action>
  <action name="actiontest_create_objects">
   <property name="text">
    <string>test create objects</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
